### ISSUE TITLE
Add Profile Picture Project

### ISSUE THREAD

**Issue by K-jais** [Vishal-raj-1/Awesome-JavaScript-Projects#1478]
This project is a custom animated feature of profile pic functionality which is there on every site nowadays. And the picture also gets saved in the local storage of the user.

Its working video,

https://user-images.githubusercontent.com/75881393/118490342-78a58900-b73b-11eb-8d93-2882c2b0f889.mp4

**Comment by K-jais** [comment]
Please @ridsuteri @991rajat @urvashi-code1255 review this issue.

**Comment by ridsuteri** [comment]
@K-jais i think this project's feature can be added to your upload image project , since most of the features are common in both the projects , what do you say?

**Comment by K-jais** [comment]
Yes sir @ridsuteri , most of the features are same then I will update the upload image project with this one.

**Comment by ridsuteri** [comment]
ok great @K-jais :)
